base/ocsp/src/main/java/com/netscape/cms/ocsp/LDAPStore.java

```diff
@@ -206,12 +207,13 @@ public void updateCRLHash(X509CertImpl caCert, X509CRLImpl crl)
 
         if (oldCRL != null) {
             if (oldCRL.getThisUpdate().getTime() >= crl.getThisUpdate().getTime()) {
-                logger.info("LDAPStore: no update, received CRL is older than current CRL");
+                logger.info("LDAPStore: no update, received CRL is not newer than current CRL");
                 return; // no update
             }
         }
-        logger.debug("Added '" + caCert.getSubjectDN() + "' into CRL hash");
+        logger.debug("LDAPStore: updateCRLHash: Added '" + caCert.getSubjectDN() + "' into CRL hash");
         mCRLs.put(caCert, crl);
+	logger.debug("LDAPStore: updateCRLHash: mCRLs size=", Integer.toString(mCRLs.size()));
```

Comment:
I think it should be written like this:
```
logger.debug("LDAPStore: updateCRLHash: mCRLs size={}", mCRLs.size());
```
or this:
```
logger.debug("LDAPStore: updateCRLHash: mCRLs size=" + mCRLs.size());
```
https://www.slf4j.org/api/org/slf4j/Logger.html#debug-java.lang.String-java.lang.Object-
https://www.slf4j.org/api/org/slf4j/helpers/MessageFormatter.html

Reply:
OH, I know my mistake now.  It's '+' not ','.  Silly me!  Thanks!  I'll make correction.